lib/shoulda/matchers/active_model/validator.rb

```diff
@@ -49,7 +49,7 @@ def has_messages?
         end
 
         def messages_description
-          if has_messages?
+          if has_messages? && !record.valid?
```

Comment:
And this change was to specifically fix https://github.com/maurogeorge/shoulda-matchers/commit/84b7175ddf85dc3ad406ac1c81335d03135268cf#diff-d7c315235a16eb29151b8251207fbd67R539, correct?